cli/verifier/verifiers.go

```diff
@@ -12,6 +12,7 @@ func AllVerifiersBeforeGenerate() []Verifier {
 func AllVerifiersAfterGenerate() []Verifier {
 	return []Verifier{
 		&NoSharedFSMount{},
+		&ImageRefValid{},
```

Comment:
Shouldn't this be a "before"-verifier? Or is your intention to check whether the images we are injecting are pinned, too?

Reply:
disadvantage after: we are pulling images that aren't pinned and that we don't want to use. If a user has many big images, this might be a difference of getting feedback after a sec or after 20 min. We should fail fast where ever possible.